test/videojs-http-streaming.test.js

```diff
@@ -261,6 +265,7 @@ QUnit.test('autoplay seeks to the live point after playlist load', function(asse
   this.clock.tick(1);
 
   openMediaSource(this.player, this.clock);
+  this.player.tech_.readyState = () => 4;
```

Comment:
IE 11 has special handing for these tests. https://github.com/videojs/http-streaming/blob/master/src/master-playlist-controller.js#L620